### ISSUE TITLE
Loss and perplexity

### ISSUE THREAD

**Issue by AmeenAli** [songlab-cal/tape#72]
hello
the loss and perplexity printed after each iteration in running tape-train is for train or validation set?

**Comment by rmrao** [comment]
This is for the training set. It is a smoothed running average of the current training metrics.
